contributor-guide/modules/ROOT/pages/contributors-faq.adoc

```diff
@@ -0,0 +1,129 @@
+
+. *What libraries have the steepest learning curve?*
++
+While all Boost libraries have their complexities, some are known to have steeper learning curves due to their advanced nature or the intricacies of the domain they address. boost:spirit[] is a parsing and generation library that uses a domain-specific embedded language (DSEL) implemented as pass:[C++] template metaprograms. It allows developers to define parsers and generators directly within code using EBNF-like syntax. However, the template-based approach and the metaprogramming techniques used can make it challenging for newcomers to grasp, especially those unfamiliar with advanced template programming or parsing theory.
++
+boost:mp11[] (Meta-Programming Library) is a powerful library for metaprogramming, providing tools for compile-time computation, type manipulation, and template metaprogramming. It allows developers to perform complex compile-time computations and transformations using a functional programming style. However, the functional programming paradigm and the intricacies of template metaprogramming can be daunting for beginners and require a solid understanding of pass:[C++] templates and meta-programming concepts.
+
+. *What libraries were the most ambitious in what they attempted to achieve?*
++
+The many notable examples include:
++
+* boost:graph[] provides a generic and efficient framework for working with graphs, making it suitable for a variety of applications in areas such as network analysis, optimization, and data visualization.
+* boost:compute[] provides abstractions for memory management, kernel execution, and data parallelism, enabling developers to harness the computational power of modern hardware for tasks such as numerical simulations, image processing, and machine learning.
+* boost:spirit[] is ambitious in its goal of providing a high-level and composable framework for parsing complex data formats and domain-specific languages entirely within pass:[C++] code, without the need for external tools or preprocessors.
+* boost:hana[] aims to simplify and modernize metaprogramming in pass:[C++], making it more accessible and powerful for developing generic libraries and applications.
+
+. *What libraries were the least ambitious technically?*
++
+The useful utilities such as boost:any[], boost:variant[], and boost:optional[] offer relatively simple functionality. Another simpler library is boost:bimap[] which provides a container for maintaining one-to-one mappings between keys and values. While bidirectional maps are a useful data structure, the functionality provided is relatively straightforward and focused on this specific use case.
+
+== Modular Boost
```

Comment:
This one is an interesting question people have. 

One problem we have is people mean different things by "modular" boost. We have content on the current website where "modular boost" simply means the libraries are in different repositories.

But now René started another discussion in the ML because he's implemented another kind of "modular Boost", where users would only need to download the libraries they want.

Since the second definition is what people seem to care about now, I think we should explain that we mean both things when we talk about modular boost (the second definition is a subset of the first definition). You could also ask René to have a look at this section.

Reply:
OK, added more explanation to the answer.